fitsio/test.py

```diff
@@ -1062,14 +1062,26 @@ def testReadFlipAxisSlice(self):
                         "Data are not the same (Expected shape: {}, actual shape: {}.".format(
                             expected_data.shape, rdata.shape))
 
-                rdata = hdu[:,130:70:6]
+                rdata = hdu[:,130:70:-6]
 
                 # Expanded by two to emulate adding one to the start value, and adding one to the calculated dimension.
                 expected_data = data[:,130:70:-6]
 
                 numpy.testing.assert_array_equal(expected_data, rdata,
                         "Data are not the same (Expected shape: {}, actual shape: {}.".format(
                             expected_data.shape, rdata.shape))
+
+                try:
+                    hdu[:,90:60:4]  # Positive step integer does not follow numpy syntax.
+                    raise AssertionError('Should raise ValueError.')
+                except ValueError as value_error:
+                    self.assertEqual('{}'.format(str(value_error)), 'slice steps must be < 0 when stop < start')
+
+                try:
+                    hdu[:,60:90:-4]
```

Comment:
with a numpy array, the above would return an empty array

Reply:
That's correct, it does return an empty array.  `numpy` wants a negative number when `x` < `y` for a slice.  Given that it will always return an empty array in this situation, I thought it better to inform the user.